README.md

```diff
@@ -209,6 +209,21 @@ user and blocks access to key chia locations.
 user with limited permissions.
 * For running in docker, you can check out @ajacobson repository for
 [chiadog-docker](https://github.com/ajacobson/chiadog-docker).
+
+## Filters
+
+Filters are also supported by adding them under the `filter` section, i.e.
+
+```yaml
+filter:
+  transaction_amount: "0.000000000002"
+```
```

Comment:
nice-to-have: Thinking about future extensibility of this, I think better structure would be:

```
handlers:
    wallet_added_coin_handler:
        filters:
            transaction_amount:  "0.000000000002"
```

then we can also use the same structure in future to allow selective enable / disable of filters like:

```
handlers:
    wallet_added_coin_handler:
        enable: False
        filters:
            transaction_amount:  "0.000000000002"
```

Important to get it as general as possible from the start, because adding fields later is much easier than migrations and backward compatibility for existing users. What do you think?